src/components/team/billing-section.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react'
+import useSubscriptionDetails from 'hooks/use-subscription-data'
+import get from 'lodash/get'
+import {format} from 'date-fns'
+import Link from 'next/link'
+import {track} from '../../utils/analytics'
+
+const BillingSection = ({
```

Comment:
This billing section displays toward the bottom of the Team Page. It lets the team owner know:
- the amount on their latest invoice
- the next billing data
- how many learners are available through their subscription